### PR TITLE
[explorer] - restore transaction amount

### PR DESCRIPTION
**Pull request by mamos-mysten**
## Description 

Add amount back to TransactionView

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by mamos-mysten** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10498** <a href="https://app.graphite.dev/github/pr/MystenLabs/sui/10498" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/MystenLabs/sui/10498?utm_source=stack-comment).

File: apps/core/src/hooks/useGetTransferAmount.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                              : '',
+                      amount:
+                          coinType === SUI_TYPE_ARG && BigInt(amount) < 0n
+                              ? BigInt(amount) + BigInt(gas ?? 0n)
+                              : BigInt(amount),
+                  }))
+                : [],
+        [balanceChanges, gas]
+    );
+    // take absolute value of the first balance change entry for display
```
Comment (Jordan-Mysten) [inline apps/core/src/hooks/useGetTransferAmount.ts:36]: Do we really only display the first balance change? Seems bad.

**Review by Jordan-Mysten — APPROVED** [review]
LGTM now, although it seems weird to only display the first transfer

File: apps/core/src/hooks/useGetTransferAmount.ts (hunk 1 above)
Comment (mamos-mysten) [reply apps/core/src/hooks/useGetTransferAmount.ts:36]: talked in person - we'll be improving this going forward

**Pull request merged**